db-init-action/action.yml

```diff
@@ -0,0 +1,50 @@
+name: "Initialize Commercial Database"
+Description: Initialize Commercial Database in parsley-services instance"
+inputs:
+  db-instance:
+    description: 'Database instance name'
+    required: true
+  db-name:
+    description: 'Database name'
+    required: true
+  gcp-workload-id-service-account-email:
+    description: "GCP service account email to assume"
+    required: true
+  gcp-workload-id-provider:
+    description: "GCP workload ID provider name"
+    required: true
+  gcp-project-id:
+    description: "GCP project of the database"
+    required: true
+  cloudsql-superuser:
+    description: "User created by Terraform in CloudSQL"
+    required: true
+  db-init-script-path:
+    description: "Path to init script relative to repository root"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v3
+    
+    - id: 'auth'
```

Comment:
So... I know I have this action/step in other composites, but I later realized that it's not really needed, or specifically the auth should've been done before this outside of a composite action at this point and gets injected as ENV vars etc. Does that make sense?